source/docs/articles/sites/errors-and-server-responses.md

```diff
@@ -10,84 +10,115 @@ Sometimes there are problems in the cloud and one of Pantheon's services is unab
 
 ### Pantheon 401 Unauthorized
-![](/source/docs/assets/images/desk_images/184676.png)  
-"This site is locked. Please contact the administrator for access." This is the default response of an HTTP Basic Auth failure after a site administrator has enabled security for an environment with a username and password. This is usually not a platform failure, as users can set environment access from their Dashboard security.  
 
-In some circumstances, a 401 can be triggered inadvertently if a site environment is locked, a user passes the HTTP auth but the site sends a 401 HTTP status code. The workaround is to disable HTTP auth security for the environment in question.
+This is the default response of an HTTP Basic Auth failure after a site administrator has enabled security for an environment with a username and password. This is usually not a platform failure, as environment access can be set from the Dashboard Security tab.  
 
-### Pantheon 403 Forbidden
-![](/source/docs/assets/images/desk_images/184677.png)  
+In some circumstances, a 401 can be triggered inadvertently if a site environment is locked, and a user passes the HTTP auth but the site sends a 401 HTTP status code. The workaround is to disable HTTP auth security for the environment in question.
 
+### Pantheon 403 Forbidden
 "Access denied to uploaded PHP files." This message is shown when a PHP file is attempted to be accessed in Valhalla, Pantheon's network file system.  
 
 Pantheon also prevents public access via the webserver to private files, .htaccess, and directory listings.
 
 ### Pantheon - 404 Unknown Site
-![](/source/docs/assets/images/desk_images/184679.png)  
-"The hostname ... is unknown. Please double-check that this is the right url. If so, make sure it matches your Dashboard's custom domain settings." This typically is shown when there is an internal routing problem or a site environment does not exist.
+"The hostname ... is unknown. Please double-check that this is the right URL. If so, make sure it matches your Dashboard's custom domain settings." This typically is shown when there is an internal routing problem or a site environment does not exist.
+
+### 502 - Upstream Header Too Big
+"Upstream sent too big header while reading response header from upstream."
+This error will occur when the payload or size of the request being sent is greater than the `fastcgi_buffer_size`. Removing additional images reduces the size of the payload being sent to the buffer for nginx to process, and will allow you to post the request. If this happens again, check to see if you are making heavy requests with a number of assets or data being passed.
+
+### 502 - Timeout/Segfault
+This issue can happen with HTTP Basic Auth and Drupal’s AJAX, as it doesn't always send the authentication if it is performing an AJAX request.
```

Comment:
@timani Is this the only situation in which this happens? Are there others? @nataliejeremy Remove the example ticket content.